css/style.css

```diff
@@ -0,0 +1,70 @@
+html,
+body {
+    background-color: #000000;
+    color: #ffff;
+    box-sizing: border-box;
+    font-family: Source Code Pro, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;;
+
```

Comment:
Empty lines should only exist when separating blocks of code. Adding linters specific to your project can easily help you filter this kind of situation and other errors :x: that can make your code unreadable :worried: .

Reply:
- [Optional]  Please when giving a review try as much to construct a friendly phrase, to give a polite comment, so it won't look like an order.
##### Good
- This is a polite suggestion: *"I think it would be better to use semantic tags"
##### Bad
-  This reads as an order: "Use semantic tags"